test/rubygems/test_gem_gem_runner.rb

```diff
@@ -109,4 +121,23 @@ def test_search_succeeds
 
     assert_empty @ui.error
   end
+
+  def test_warn_on_outdated
+    args = %w[info]
+    singleton_gem_class.class_eval do
+      def latest_rubygems_version
+        Gem::Version.new "99999999.0.0"
+      end
+    end
+
+    use_ui @ui do
+      @runner.run(args)
+    end
+  end
```

Comment:
It looks like this is basically just testing that it successfully runs. Should this be checking the output at all?